app/c2/gist.py

```diff
@@ -0,0 +1,138 @@
+import json
+import uuid
+import aiohttp
+import re
+from base64 import b64encode
+
+from app.objects.c_c2 import C2
+
+
+class Gist(C2):
+
+    def __init__(self, module_info):
+        self.name = 'Gist'
+        self.key = module_info['config']['key']
+        self.c2_type = module_info['c2_type']
+
+    def encode_config_info(self):
+        """
+        Returns one of the GIST API keys to be encoded into the agent
+        :return: GIST api key
+        """
+        return self.key
+
+    async def get_results(self):
+        """
+        Retrieve all GIST posted results for a particular api key
+        :return:
```

Comment:
This says "for a particular API key"... is the plan to eventually pass through the API key as an argument here?

Reply:
I don't think that's the plan. The docstring is misleading. I'll reword it.